security/proto/nodeagent_service.proto

```diff
@@ -21,6 +21,12 @@ option go_package="istio_v1_auth";
 
 service NodeAgentService {
```

Comment:
Can you add a short comment on the purpose this service?

Reply:
done, thanks